### PR TITLE
[IMP] General: Google Oauth Docs edit

### PR DESCRIPTION
**Pull request by tiku-odoo**
Targets 14.0 onward. 

Google Oauth Changes to Sign-On, Calendar, Gmail
Changes: update to match new interface and remove second person, add details on Internal and External User Types.

Google Calendar Sync doc
https://www.odoo.com/web#id=3246409&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

[Calendar] Google Sync warning
https://www.odoo.com/web#id=3000531&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

Google Calendar Sync Videos
https://www.odoo.com/web#id=2859258&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

Google Sign-In Authentication User Type
https://www.odoo.com/web#id=3247706&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

[auth] rewrite google.rst
https://www.odoo.com/web#id=3252200&cids=3&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@odoo/us-doc-review This set of docs is ready for your review when you have some time. Thanks in advance for your time on these 👍

File: content/applications/general/calendars/google/google_calendar_credentials.rst [hunk 1]
```diff
@@ -45,58 +46,57 @@ Calendar API` from the search results. Click :guilabel:`Enable`.
...
-Create credentials
-------------------
+OAuth consent screen
+--------------------
 
-Now that you have created your API project and enabled the Google Calendar API, you need to create
-credentials. Begin by clicking :guilabel:`Create Credentials`. Google will then guide you through
-four steps to create your API credentials.
+Now that the API project has been created OAuth should be configured. Click on :guilabel:`OAuth
+consent` in the left menu and then select the :guilabel:`User Type`.
```
Comment (ksc-odoo) [inline content/applications/general/calendars/google/google_calendar_credentials.rst:53]: Now that the API project has been created, OAuth should be configured. To do that, click on :guilabel:`OAuth
consent` in the left menu, and then select the :guilabel:`User Type`.

File: content/applications/general/calendars/google/google_calendar_credentials.rst [hunk 2]
```diff
@@ -45,58 +46,57 @@ Calendar API` from the search results. Click :guilabel:`Enable`.
...
 
-- In the first step, :guilabel:`Credential Type`, select the :guilabel:`Google Calendar API` and
-  :guilabel:`User Data` options. Then, click :guilabel:`Next`.
+.. warning::
+   Personal Gmail Accounts are only allowed to be :guilabel:`External` User Type, which means
+   Google may require an approval or for :guilabel:`Scopes` to be added on. Using a Google WorkSpace
+   account allows for :guilabel:`Internal` User Type. It should also be noted that while the API
+   connection is in the :guilabel:`External` :guilabel:`Testing` mode then no approval is necessary
+   from Google. User limits in this :guilabel:`Testing` mode is set to 100 users.
```
Comment (ksc-odoo) [inline content/applications/general/calendars/google/google_calendar_credentials.rst:60]: ...Google may require an approval, or for :guilabel:`Scopes` to be added on. Using a Google WorkSpace
   account allows for :guilabel:`Internal` User Type. It should also be noted that, while the API
   connection is in the :guilabel:`External` :guilabel:`Testing` mode, then no approval is necessary
   from Google. User limits in this :guilabel:`Testing` mode is set to 100 users.

File: content/applications/general/calendars/google/google_calendar_credentials.rst [hunk 3]
```diff
@@ -45,58 +46,57 @@ Calendar API` from the search results. Click :guilabel:`Enable`.
...
-   :alt: Select Google Calendar API and User Data for the Credential Type.
+In the second step, :guilabel:`OAuth Consent Screen`, type *Odoo* in the :guilabel:`App name`
+field, select the email address for the :guilabel:`User support email` field, and type the email
+address for the :guilabel:`Developer contact information` section. Then, click :guilabel:`Save
+and Continue`.
+
+Skip the third step, :guilabel:`Scopes`, by clicking :guilabel:`Save and Continue`.
+
+Next, if continuing in testing mode (External), add the email addresses being configured under the
+:guilabel:`Test users` step by clicking on :guilabel:`Add Users` and then the :guilabel:`Save and
```
Comment (ksc-odoo) [inline content/applications/general/calendars/google/google_calendar_credentials.rst:70]: ...step, by clicking :guilabel:`Add Users`, and then the...

File: content/applications/general/calendars/google/google_calendar_credentials.rst [hunk 4]
```diff
@@ -45,58 +46,57 @@ Calendar API` from the search results. Click :guilabel:`Enable`.
...
+Continue` button. A summary of the :guilabel:`App registration` appears.
 
-- In the second step, :guilabel:`OAuth Consent Screen`, type *Odoo* in the :guilabel:`App name`
-  field, select your email address for the :guilabel:`User support email` field, and type your email
-  address for the :guilabel:`Developer contact information` section. Then, click :guilabel:`Save
-  and Continue`.
+Finally, scroll to the bottom and click on :guilabel:`Back to Dashboard`.
```
Comment (ksc-odoo) [inline content/applications/general/calendars/google/google_calendar_credentials.rst:73]: Finally, scroll to the bottom, and click on :guilabel:`Back to Dashboard`.

File: content/applications/general/calendars/google/google_calendar_credentials.rst [hunk 5]
```diff
@@ -45,58 +46,57 @@ Calendar API` from the search results. Click :guilabel:`Enable`.
...
 
-- Skip the third step, :guilabel:`Scopes`, by clicking :guilabel:`Save and Continue`.
+Now the OAuth consent has been configured and it's time to create credentials.
```
Comment (ksc-odoo) [inline content/applications/general/calendars/google/google_calendar_credentials.rst:75]: Now, the OAuth consent has been configured, and it's time to create credentials.

File: content/applications/general/calendars/google/google_calendar_credentials.rst [hunk 6]
```diff
@@ -45,58 +46,57 @@ Calendar API` from the search results. Click :guilabel:`Enable`.
...
+
+Create credentials
+------------------
 
-- In the last step, :guilabel:`OAuth Client ID`, select :guilabel:`Website application` for the
-  :guilabel:`Application Type` field and type *My Odoo Database* for the :guilabel:`Name`.
+The **Client ID** and the **Client Secret** are both needed to connect Google Calendar to Odoo. This
+is the last step in the Google console, begin by clicking :guilabel:`Credentials` in the left menu.
+Then click :guilabel:`Create Credentials` and select :guilabel:`OAuth client ID` Google will open a
+guide to create credentials.
```
Comment (ksc-odoo) [inline content/applications/general/calendars/google/google_calendar_credentials.rst:83]: The **Client ID** and the **Client Secret** are both needed to connect Google Calendar to Odoo. This
is the last step in the Google console. Begin by clicking :guilabel:`Credentials` in the left menu.
Then, click :guilabel:`Create Credentials` , select :guilabel:`OAuth client ID`, and Google will open a
guide to create credentials.

File: content/applications/general/calendars/google/google_calendar_credentials.rst [hunk 7]
```diff
@@ -45,58 +46,57 @@ Calendar API` from the search results. Click :guilabel:`Enable`.
...
+
+Under :guilabel:`OAuth Client ID`, select :guilabel:`Website application` for the
+:guilabel:`Application Type` field and type *My Odoo Database* for the :guilabel:`Name`.
```
Comment (ksc-odoo) [inline content/applications/general/calendars/google/google_calendar_credentials.rst:86]: ...field, and type...

File: content/applications/general/calendars/google/google_calendar_credentials.rst [hunk 8]
```diff
@@ -121,15 +124,28 @@ Google Calendar with Odoo.
    :alt: Click the Google sync button in Odoo Calendar to sync Google Calendar with Odoo.
 
 .. note::
-   The first time you sync your Google Calendar with Odoo, the page will redirect to your Google
-   Account. Click :guilabel:`OK` and :guilabel:`Allow` to authorize Odoo to access Google Calendar.
+   The first time syncing Google Calendar with Odoo, the page will redirect to the Google
+   Account. Select the :menuselection:`Email Account` that should have access -->
+   :menuselection:`Continue` (should the app be unverifed) --> :menuselection:`Continue` (to give
+   permission for the transfer of data)
```
Comment (ksc-odoo) [inline content/applications/general/calendars/google/google_calendar_credentials.rst:130]: While synching Google Calendar with Odoo for the first time, the page will redirect to the Google
   Account. From there, select the :menuselection:`Email Account` that should have access -->
   :menuselection:`Continue` (should the app be unverifed) --> :menuselection:`Continue` (to give
   permission for the transfer of data).

**Review by ksc-odoo — COMMENTED** [review]
Hey @tiku-odoo Just finished a peer review on this one - Great work! Let me know if you have any questions or need clarification on anything.

**Review by bouvyd — APPROVED** [review]

File: content/administration/maintain/google_oauth.rst [hunk 9]
```diff
@@ -38,10 +34,22 @@ you are using a personal Gmail account, then leave the :guilabel:`Location` as `
...
 
-Click on :guilabel:`Create` to finish this step. On the next screen, :guilabel:`User Type` options,
-select the :guilabel:`External` option, and then click on :guilabel:`Create` again, which will
+Click on :guilabel:`Create` to finish this step.
+
+OAuth consent screen
+--------------------
+
+Click on :guilabel:`OAuth consent screen` in the left menu, :guilabel:`User Type` options,
+select the :guilabel:`User Type` option, and then click on :guilabel:`Create` again, which will
```
[the remark this answers is not in the document]
Comment (tiku-odoo) [reply content/administration/maintain/google_oauth.rst]: Thanks. This process is a bit complicated, as it goes through a couple of pages. I'm going to keep it as is for now. Thanks.

**Comment by tiku-odoo** [comment]
@samueljlieber These docs are ready for your technical review. 

Thanks in advanced for your help on these.

**Comment by jcs-odoo** [comment]
Hi @samueljlieber 

I think your last force-push is incorrect as it moves the calendar docs back to general/calendars, as introduced in #4032

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
This pr currently reverts the structure change made in #4032 . Is this unintentional? The conflict during rebase should be handled the other way around (the pr adapts to the current structure of the doc)

File: content/applications/general/calendars/google/google_calendar_credentials.rst [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+
+.. image:: google_calendar_credentials/sync-google.png
+   :align: center
+   :alt: Click the Google sync button in Odoo Calendar to sync Google Calendar with Odoo.
+
+.. note::
+   When syncing Google Calendar with Odoo for the first time, the page will redirect to the Google
+   Account. From there, select the :menuselection:`Email Account` that should have access -->
+   :menuselection:`Continue` (should the app be unverifed) --> :menuselection:`Continue` (to give
+   permission for the transfer of data).
```
Comment (samueljlieber) [inline content/applications/general/calendars/google/google_calendar_credentials.rst:134]: The phrasing and formatting within this paragraph makes supporting the proper :menuselection: syntax difficult, I'm suggesting we rephrase it to simply use :guilabels:

```
When syncing Google Calendar with Odoo for the first time, the page will redirect to the Google Account. From there, select the :guilabel:`Email Account` that should have access, then select :guilabel:`Continue` (should the app be unverifed), and finally select :guilabel:`Continue` (to give permission for the transfer of data)`.
```

File: content/applications/general/auth/google.rst [hunk 11]
```diff
@@ -44,16 +44,29 @@ OAuth consent screen
...
+   .. warning::
+      Personal Gmail Accounts are only allowed to be :guilabel:`External` User Type, which means
+      Google may require an approval, or for :guilabel:`Scopes` to be added on. Using a Google
+      WorkSpace account allows for :guilabel:`Internal` User Type. It should also be noted, that
+      while the API connection is in the :guilabel:`External` :guilabel:`Testing` mode, then no
+      approval is necessary from Google. User limits in this :guilabel:`Testing` mode is set to 100
+      users.
+
+#. Fill out the required details and domain info, then click on :guilabel:`Save and Continue`.
+#. On the **Scopes** page, leave all fields as is, and click on :guilabel:`Save and Continue`.
```
Comment (samueljlieber) [inline content/applications/general/auth/google.rst:64]: lets italicize _Scopes_ instead of bold here

File: content/applications/general/auth/google.rst [hunk 12]
```diff
@@ -64,24 +77,25 @@ Credentials
...
+#. Click on :guilabel:`Create Credentials`, and select :guilabel:`OAuth client ID`.
 
    .. image:: google/client-id.png
       :align: center
-      :alt: Oauth client id selection
+      :alt: Oauth client id selection.
 
-#. Select **Web Application** as the Application type. Now configure the allowed pages on which you
-   will be redirected.
+#. Select :guilabel:`Web Application` as the Application type. Now, configure the allowed pages on
```
Comment (samueljlieber) [inline content/applications/general/auth/google.rst:88]: lets :guilabel: _Application type_ here

File: content/administration/maintain/google_oauth.rst [hunk 13]
```diff
@@ -38,10 +38,22 @@ you are using a personal Gmail account, then leave the :guilabel:`Location` as `
...
+Click on :guilabel:`OAuth consent screen` in the left menu, under :guilabel:`User Type` options,
+select the appropriate :guilabel:`User Type`, and then click on :guilabel:`Create` again, which will
 finally navigate to the :guilabel:`Edit app registration` page.
 
+.. warning::
+   Personal Gmail Accounts are only allowed to be :guilabel:`External` User Type, which means
+   Google may require an approval, or for :guilabel:`Scopes` to be added on. Using a Google
+   WorkSpace account allows for :guilabel:`Internal` User Type. It should also be noted, that while
+   the API connection is in the :guilabel:`External` :guilabel:`Testing` mode, then no approval is
+   necessary from Google. User limits in this :guilabel:`Testing` mode is set to 100 users.
```
Comment (samueljlieber) [inline content/administration/maintain/google_oauth.rst:55]: italicize and lowercase *testing* mode

File: content/administration/maintain/google_oauth.rst [hunk 14]
```diff
@@ -56,23 +68,22 @@ Edit app registration
 - Next, click on the :guilabel:`Save and Continue` button.
 - Then, skip the :guilabel:`Scopes` page by scrolling to the bottom and clicking on
   :guilabel:`Save and Continue`.
-- Next, add the email being configured under the :guilabel:`Test users` step by clicking on
-  :guilabel:`Add Users` and then the :guilabel:`Save and Continue` button. A summary of the
-  :guilabel:`App registration` appears.
+- Next, if continuing in testing mode (External), add the email addresses being configured under the
+  :guilabel:`Test users` step, by clicking on :guilabel:`Add Users`, and then the :guilabel:`Save and
```
Comment (samueljlieber) [inline content/administration/maintain/google_oauth.rst:72]: line needs to break earlier at `Save`

**Comment by samueljlieber** [comment]
> Fixed merge conflict in [f9a7849](https://github.com/odoo/documentation/commit/f9a7849ce6735d3912e4226ecefc3fe3a41c0ba0).

Hi @jcs-odoo Ah - I see I may have misunderstood the changes in this PR when correcting the merge conflict. Let me review with @tiku-odoo and @StraubCreative and I will correct this.

**Comment by jcs-odoo** [comment]
> > Hi @samueljlieber
> > I think your last force-push is incorrect as it moves the calendar docs back to general/calendars. The docs were moved to productivity/calendar, as introduced in #4032
> 
> > This pr currently reverts the structure change made in #4032 . Is this unintentional? The conflict during rebase should be handled the other way around (the pr adapts to the current structure of the doc)
> 
> Hi @jcs-odoo Ah - I see I may have misunderstood the changes in this PR when correcting the merge conflict. Let me review with @tiku-odoo and @StraubCreative and I will correct this.

Thanks 😊

**Comment by samueljlieber** [comment]
Hi @tiku-odoo & @jcs-odoo 👋

I reverted the changes I made in my merge conflict commit (https://github.com/odoo/documentation/commit/f9a7849ce6735d3912e4226ecefc3fe3a41c0ba0) and applied my [technical changes](https://github.com/odoo/documentation/pull/3965#pullrequestreview-1388309097) on the correct structure, my apologies for the mix-up!

I received approval of the technical changes from @tiku-odoo externally, so I'm ready to hand this off to @StraubCreative for a final review.

Thank you 🙂

**Review by samueljlieber — APPROVED** [review]

File: content/administration/maintain/google_oauth.rst [hunk 15]
```diff
@@ -38,10 +38,22 @@ you are using a personal Gmail account, then leave the :guilabel:`Location` as `
...
+Click on :guilabel:`OAuth consent screen` in the left menu, under :guilabel:`User Type` options,
+select the appropriate :guilabel:`User Type`, and then click on :guilabel:`Create` again, which will
 finally navigate to the :guilabel:`Edit app registration` page.
 
+.. warning::
+   Personal Gmail accounts are only allowed to be :guilabel:`External` User Type, which means Google
+   may require an approval, or for :guilabel:`Scopes` to be added on. Using a Google WorkSpace
+   account allows for :guilabel:`Internal` User Type. It should also be noted, that while the API
+   connection is in the :guilabel:`External` :guilabel:`Testing` mode, then no approval is necessary
+   from Google. User limits in this *testing* mode is set to 100 users.
```
Comment (StraubCreative) [inline content/administration/maintain/google_oauth.rst:55]: @tiku-odoo can you take another look here around **guilabel** usage?

It *seems* that some of these should be in **bold** or _italics_ since they appear to:
- be referred to theoretically, and not through active UI instruction
- not be the actual names of headings, buttons, fields, values, etc. on the UI

As well, check capitalization since none of these appear to be proper nouns or product names, i.e. is **External** User Type supposed to be capitalized? Is this unique to Google?

I'm not saying there's a clear right answer for every case, however with the way this is written it's difficult to decipher. You might benefit from changing the language around slightly to _force_ specific RST use more clearly.

File: content/administration/maintain/google_oauth.rst [hunk 16]
```diff
@@ -56,26 +68,25 @@ Edit app registration
 - Next, click on the :guilabel:`Save and Continue` button.
 - Then, skip the :guilabel:`Scopes` page by scrolling to the bottom and clicking on
   :guilabel:`Save and Continue`.
-- Next, add the email being configured under the :guilabel:`Test users` step by clicking on
-  :guilabel:`Add Users` and then the :guilabel:`Save and Continue` button. A summary of the
-  :guilabel:`App registration` appears.
+- Next, if continuing in testing mode (External), add the email addresses being configured under the
```
Comment (StraubCreative) [inline content/administration/maintain/google_oauth.rst:71]: Don't see the benefit of adding this part?
> Next, if continuing in testing mode (External)

The previous version seemed clearer because it was active instruction, and not an "if" scenario...

File: content/administration/maintain/google_oauth.rst [hunk 17]
```diff
@@ -56,26 +68,25 @@ Edit app registration
...
 - Finally, scroll to the bottom and click on :guilabel:`Back to Dashboard`.
 
 Create Credentials
 ------------------
 
-Now that you have set up the project, you need to create credentials, which includes the
-:guilabel:`Client ID` and :guilabel:`Client Secret`. First, click on :guilabel:`Credentials` in the
-left sidebar menu.
+Now that the project is set up, credentials should be created, which includes the :guilabel:`Client
+ID` and :guilabel:`Client Secret`. First, click on :guilabel:`Credentials` in the left sidebar menu.
```
Comment (StraubCreative) [inline content/administration/maintain/google_oauth.rst:80]: Formatting...
```suggestion
Now that the project is set up, the next step is to generate credentials, which includes the *client
ID* and *client secret* values. Get started by clicking on :menuselection:`Credentials` in the left sidebar menu.
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @tiku-odoo 

Seeing a lot of RST tag use and capitalization issues here.

There are spots where passive language is hurting your ability to communicate clearly and also justify the use of specific RST tags. As a result there's lots of gray area for how to format things...

Ideally, there's no question around formatting if the language is clear/assertive and we don't unnecessarily refer to UI elements unless using active instruction. For example, why are we talking about field names/values if we're not even on the screen yet?

Can you take another look please?

Thanks!

File: content/administration/maintain/google_oauth.rst (hunk 16 above)
Comment (tiku-odoo) [reply content/administration/maintain/google_oauth.rst:71]: This is a common mistake made during configuration and Google is currently changing up requirements so I was avoiding which Mode to put the configuration. I think it is important to include this part.

**Comment by samueljlieber** [comment]
Hi @tiku-odoo, I pushed your changes to the docs in https://github.com/odoo/documentation/commit/ad7795f83d540eeed1d26539f274568bcd6fb191. Please review and tag ZST when you are ready 🙂

To pull down these changes, please use `git pull --rebase` - let me know if you have any issues doing so!

**Comment by tiku-odoo** [comment]
@StraubCreative This doc is ready for another look by you. I made changes to the GUI Labels. They do appear capitalized in the UI. I've used italics in many of the cases. Thanks for your suggestions.

File: content/administration/maintain/google_oauth.rst [hunk 18]
```diff
@@ -158,38 +176,36 @@ populate.
...
+search for ``Gmail`` or ``Google`` and upgrade the :guilabel:`Google Gmail` module. Finally, click
+on the three dots on the upper right of the module and select :guilabel:`Upgrade`.
 
 Application Type
 ----------------
 
-When creating the :guilabel:`Credentials` (OAuth Client ID and Client Secret), if
-:guilabel:`Desktop App` is selected for the :guilabel:`Application Type`, an
-:guilabel:`Authorization Error` appears.
+When creating the *credentials* (OAuth Client ID and Client Secret), if :guilabel:`Desktop App` is
```
Comment (StraubCreative) [inline content/administration/maintain/google_oauth.rst:199]: Close
```suggestion
When creating the credentials (OAuth *Client ID* and *Client Secret*), if :guilabel:`Desktop App` is
```

File: content/administration/maintain/google_oauth.rst [hunk 19]
```diff
@@ -158,38 +176,36 @@ populate.
...
    :alt: Error 400 Redirect URI Mismatch.
 
 To correct this error, delete the credentials already created and create new credentials, selecting
-:guilabel:`Web Application` for the :guilabel:`Application Type`. Then, under
-:guilabel:`Authorized redirect URIs`, click :guilabel:`ADD URI` and type:
+:guilabel:`Web Application` for the :guilabel:`Application Type`. Then, under :guilabel:`Authorized
+redirect URIs`, click :guilabel:`ADD URI` and type:
 `https://yourdbname.odoo.com/google_gmail/confirm` in the field.
 
 .. note::
```
Comment (StraubCreative) [inline content/administration/maintain/google_oauth.rst:211]: Matching the earlier instruction, deleting the note here.
```suggestion
`https://yourdbname.odoo.com/google_gmail/confirm` in the field, being sure to replace *yourdbname*
in the URL with the Odoo database name.
```

File: content/applications/general/auth/google.rst [hunk 20]
```diff
@@ -44,16 +45,30 @@ OAuth consent screen
...
+      :alt: Choice of a user type in OAuth consent.
 
-#. Fill out your details and domain info, then click on *Save and Continue*.
-#. On the **Scopes** page, leave all fields as is, and click on *Save and Continue*.
+   .. warning::
+      Personal Gmail Accounts are only allowed to be *External* User Type, which means Google may
+      require an approval, or for *Scopes* to be added on. Using a Google WorkSpace account allows
+      for *Internal* User Type. It should also be noted, that while the API connection is in the
+      *External* testing mode, then no approval is necessary from Google. User limits in this
+      testing mode is set to 100 users.
```
Comment (StraubCreative) [inline content/applications/general/auth/google.rst:62]:
```suggestion
      *Personal* Gmail Accounts are only allowed to be **External** User Type, which means Google
      may require an approval, or for *Scopes* to be added on. However, using a *Google WorkSpace*
      account allows for **Internal** User Type to be used.

      Note, as well, that while the API connection is in the *External* testing mode, then no
      approval is necessary from Google. User limits in this testing mode is set to 100 users.
```

File: content/applications/general/auth/google.rst [hunk 21]
```diff
@@ -2,38 +2,39 @@
 Google Sign-In Authentication
```
Comment (StraubCreative) [inline content/applications/general/auth/google.rst:2]: For this page I'm not sure how helpful the numbered list instructions are vs. paragraph form.
Not a block for the review, however maybe we can entertain restructuring at some point.

File: content/applications/general/auth/google.rst [hunk 22]
```diff
@@ -44,16 +45,30 @@ OAuth consent screen
...
+      *External* testing mode, then no approval is necessary from Google. User limits in this
+      testing mode is set to 100 users.
+
+#. Fill out the required details and domain info, then click on :guilabel:`Save and Continue`.
+#. On the :menuselection:`Scopes` page, leave all fields as is, and click on :guilabel:`Save and
+   Continue`.
+#. Next, if continuing in testing mode (*External*), add the email addresses being configured under
+   the :guilabel:`Test users` step by clicking on :guilabel:`Add Users`, and then the
+   :guilabel:`Save and Continue` button. A summary of the app registration appears.
+
```
Comment (StraubCreative) [inline content/applications/general/auth/google.rst:70]:
```suggestion
```

File: content/applications/general/auth/google.rst [hunk 23]
```diff
@@ -64,24 +79,24 @@ Credentials
...
+   In order to achieve this, in the :guilabel:`Authorized redirect URIs` field, enter the database's
+   domain immediately followed by ``/auth_oauth/signin``. For example:
+   ``https://mydomain.odoo.com/auth_oauth/signin``, then click on :guilabel:`Create`.
 
-   .. image:: google/create-client-id.png
-      :align: center
-      :alt: Creating oauth client id
+#. Now the *OAuth client* has been created. A screen will appear with the :guilabel:`Client ID` and
+   :guilabel:`Client Secret`. Copy (or write down) the :guilabel:`Client ID`, as it will be
+   necessary for the configuration in Odoo, which will be covered in the following steps.
```
Comment (StraubCreative) [inline content/applications/general/auth/google.rst:99]:
```suggestion
#. Now that the *OAuth client* has been created, a screen will appear with the :guilabel:`Client ID`
   and :guilabel:`Client Secret`. Copy the :guilabel:`Client ID` for later, as it will be necessary
   for the configuration in Odoo, which will be covered in the following steps.
```

File: content/applications/general/auth/google.rst [hunk 24]
```diff
@@ -93,47 +108,53 @@ Google Authentication on Odoo
...
+   :guilabel:`Client ID` with the key from the Google API Dashboard, and :guilabel:`Save`.
 
    .. image:: google/odoo-client-id.png
       :align: center
-      :alt: Filling out the client id in Odoo settings
+      :alt: Filling out the client id in Odoo settings.
+
+   .. Note::
+     Google OAuth2 configuration can also be accessed by clicking on :guilabel:`OAuth Providers`
+     under the :guilabel:`OAuth Authentication` heading in :menuselection:`Integrations`.
```
Comment (StraubCreative) [inline content/applications/general/auth/google.rst:140]:
```suggestion
   .. note::
      Google OAuth2 configuration can also be accessed by clicking on :guilabel:`OAuth Providers`
      under the :guilabel:`OAuth Authentication` heading in :menuselection:`Integrations`.
```

File: content/applications/general/auth/google.rst [hunk 25]
```diff
@@ -93,47 +108,53 @@ Google Authentication on Odoo
...
       :align: center
-      :alt: Reset password screen with "Log in with Google" button
+      :alt: Reset password screen with "Log in with Google" button.
+
 
-Existing users must :ref:`reset their password <users/reset-password>` to access the *reset
-password* page, while new users can directly click on *Log in with Google* instead of choosing a new
-password.
+Existing users must :ref:`reset their password <users/reset-password>` to access the
+:menuselection:`reset password` page, while new users can directly click on :guilabel:`Log in with
```
Comment (StraubCreative) [inline content/applications/general/auth/google.rst:156]:
```suggestion
:menuselection:`Reset Password` page, while new users can directly click on :guilabel:`Log in with
```

File: content/applications/productivity/calendar/google.rst [hunk 26]
```diff
@@ -12,91 +12,94 @@ in both directions). This integration helps organize your schedule so you never
...
+:guilabel:`OAuth consent` in the left menu and then select the :guilabel:`User Type`.
 
-- In the first step, :guilabel:`Credential Type`, select the :guilabel:`Google Calendar API` and
-  :guilabel:`User Data` options. Then, click :guilabel:`Next`.
+.. warning::
+   Personal Gmail Accounts are only allowed to be *External* User Type, which means Google may
+   require an approval, or for *Scopes* to be added on. Using a Google WorkSpace account allows for
+   *Internal* User Type. It should also be noted, that while the API connection is in the *External*
+   testing mode, then no approval is necessary from Google. User limits in this testing mode is set
+   to 100 users.
```
Comment (StraubCreative) [inline content/applications/productivity/calendar/google.rst:64]:
```suggestion
.. warning::
   *Personal* Gmail Accounts are only allowed to be **External** User Type, which means Google may
   require an approval, or for *Scopes* to be added on. However, using a *Google WorkSpace* account
   allows for **Internal** User Type to be used.

   Note, as well, that while the API connection is in the *External* testing mode, then no approval is
   necessary from Google. User limits in this testing mode is set to 100 users.
```

File: content/applications/productivity/calendar/google.rst [hunk 27]
```diff
@@ -12,91 +12,94 @@ in both directions). This integration helps organize your schedule so you never
...
 
-.. image:: google/credential-type.png
-   :align: center
-   :alt: Select Google Calendar API and User Data for the Credential Type.
+In the second step, :guilabel:`OAuth Consent Screen`, type *Odoo* in the :guilabel:`App name`
+field, select the email address for the :guilabel:`User support email` field, and type the email
+address for the :guilabel:`Developer contact information` section. Then, click :guilabel:`Save
+and Continue`.
```
Comment (StraubCreative) [inline content/applications/productivity/calendar/google.rst:69]: this is string input so gets `pre` formatting
also, 100th char line breaks
```suggestion
In the second step, :guilabel:`OAuth Consent Screen`, type `Odoo` in the :guilabel:`App name` field,
select the email address for the :guilabel:`User support email` field, and type the email address
for the :guilabel:`Developer contact information` section. Then, click :guilabel:`Save and
Continue`.
```

File: content/applications/productivity/calendar/google.rst [hunk 28]
```diff
@@ -12,91 +12,94 @@ in both directions). This integration helps organize your schedule so you never
...
 
-After successfully creating a new API project, enabling the Google Calendar API, and generating the
-Google Calendar API credentials, you should now have a Client ID and Client Secret.
+Create credentials
+------------------
 
-Client ID & Client Secret
--------------------------
+The *Client ID* and the *Client Secret* are both needed to connect Google Calendar to Odoo. This
+is the last step in the Google console. Begin by clicking :guilabel:`Credentials` in the left menu.
```
Comment (StraubCreative) [inline content/applications/productivity/calendar/google.rst:85]:
```suggestion
The *Client ID* and the *Client Secret* are both needed to connect Google Calendar to Odoo. This is
the last step in the Google console. Begin by clicking :guilabel:`Credentials` in the left menu.
```

File: content/applications/productivity/calendar/google.rst [hunk 29]
```diff
@@ -12,91 +12,94 @@ in both directions). This integration helps organize your schedule so you never
...
-to :menuselection:`API & Services --> Credentials --> OAuth 2.0 Client IDs`.
+Under :menuselection:`Create OAuth Client ID`, select :guilabel:`Website application` for the
+:guilabel:`Application Type` field, and type `My Odoo Database` for the :guilabel:`Name`.
 
-Next, locate the credentials you just created for the Google Calendar API. Then, click on
-:guilabel:`Edit OAuth Client` (the pencil icon). The page will redirect to the edit page, where you
-can view the Client ID and the Client Secret.
+- Under the ::menuselection:`Authorized JavaScript Origins` section, click :guilabel:`+ Add URI` and
+  type the company's Odoo full :abbr:`URL (Uniform Resource Locator)` address.
+- Under the ::menuselection:`Authorized redirect URIs` section, click :guilabel:`+ Add URI` and type
```
Comment (StraubCreative) [inline content/applications/productivity/calendar/google.rst:94]: menuselection has too many colons (::)
these are guilabels anyway.
```suggestion
- Under the :guilabel:`Authorized JavaScript Origins` section, click :guilabel:`+ Add URI` and
  type the company's Odoo full :abbr:`URL (Uniform Resource Locator)` address.
- Under the :guilabel:`Authorized redirect URIs` section, click :guilabel:`+ Add URI` and type
```

File: content/applications/productivity/calendar/google.rst [hunk 30]
```diff
@@ -109,27 +112,43 @@ next to :guilabel:`Google Calendar`.
...
 
-Finally, open the Calendar module in Odoo and click on the :guilabel:`Google` sync button to sync
-Google Calendar with Odoo.
+Sync calendar in Odoo
+=====================
+
+Finally, open the :menuselection:`Calendar` app in Odoo and click on the :guilabel:`Google` sync
+button to sync Google Calendar with Odoo.
 
 .. image:: google/sync-google.png
```
Comment (StraubCreative) [inline content/applications/productivity/calendar/google.rst:125]: Is this screenshot only for 16? @tiku-odoo

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @tiku-odoo 

Nice cleanup here.
I have change requests below which I'll be pushing up momentarily for merge.

There are a few questions in my review which we might want to consider for different versions of these docs. As well, I'm not convinced the "bullet-list style" works (vs. a narrative guide) so wherever that's the dominant style on a doc, a rewrite might be warranted there.

File: content/applications/productivity/calendar/google.rst (hunk 30 above)
Comment (tiku-odoo) [reply content/applications/productivity/calendar/google.rst:126]: This should be relevant for all versions @StraubCreative

**Review by StraubCreative — APPROVED** [review]
@robodoo r+
